### DISCUSSION TITLE
SPI1 DWM1000

### DISCUSSION THREAD

**Discussion by hhl-16** [stm32duino/Arduino_Core_STM32#1917] [Q&A]
hi guys
i am using stm32f407vet6 spi with dwm1000
when i use PB15 PB14 PB13 as SPI pin connected to dwm1000
I looked at the documentation and it was SPI2, I run the example BasicSender  and dwm1000 works fine
https://github.com/thotro/arduino-dw1000

when I use SPI1 as PA5 PA6 PA7 it doesn't work
thanks

**Comment by fpistm** [comment]
Hi,
Could you be more precise. Which target you used? How you change SPI pins?
Library you used seems using the default SPI instance.
See https://github.com/stm32duino/wiki/wiki/API#change-default-spi-instance-pins to change default SPI pins.
